test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ClientYamlTestClient.java

```diff
@@ -57,7 +57,11 @@
 public class ClientYamlTestClient {
     private static final ESLogger logger = Loggers.getLogger(ClientYamlTestClient.class);
     //query_string params that don't need to be declared in the spec, they are supported by default
-    private static final Set<String> ALWAYS_ACCEPTED_QUERY_STRING_PARAMS = Sets.newHashSet("pretty", "source", "filter_path");
+    private static final Set<String> ALWAYS_ACCEPTED_QUERY_STRING_PARAMS = Sets.newHashSet("pretty",
+                                                                                            "source",
+                                                                                            "filter_path",
+                                                                                            "filter_path_include",
+                                                                                            "filter_path_exclude");
```

Comment:
you should sync up with @elastic/es-clients to make sure that they make the same change in their clients/runners if needed. Otherwise their runners may barf as some parameters are provided although they are not listed in the api spec.